css/css-ruby/inheritance.html

```diff
@@ -17,6 +17,8 @@
 <script>
 assert_inherited('ruby-align', 'space-around', 'center');
 assert_inherited('ruby-position', 'alternate', 'under');
+assert_inherited('ruby-merge', 'separate', 'merge');
```

Comment:
?

Reply:
It was just a drive-by fix since we weren't testing inheritance: https://drafts.csswg.org/css-ruby-1/#collapsed-ruby , I can clarify the commit message